js/renovation/ui/common/__tests__/widget.test.tsx

```diff
@@ -544,6 +546,45 @@ describe('Widget', () => {
         });
       });
 
+      describe('windowResizeEffect', () => {
+        afterEach(() => {
+          jest.clearAllMocks();
+        });
+
+        it('should subscribe to windowResizeEffect event', () => {
+          const onDimensionChanged = jest.fn();
+          const widget = new Widget({ onDimensionChanged });
+
+          const dispose = widget.windowResizeEffect() as DisposeEffectReturn;
+
+          expect(resizeCallbacks.add).toBeCalledTimes(1);
+          expect(resizeCallbacks.remove).toBeCalledTimes(0);
+
+          dispose?.();
+        });
+
+        it('should return unsubscribe callback', () => {
+          const onDimensionChanged = jest.fn();
+          const widget = new Widget({ onDimensionChanged });
+
+          const dispose = widget.windowResizeEffect() as DisposeEffectReturn;
+
+          dispose?.();
+
+          expect(resizeCallbacks.remove).toBeCalledTimes(1);
+          const callbackPassedToAdd = (resizeCallbacks as any).add.mock.calls[0][0];
+          const callbackPassedToRemove = (resizeCallbacks as any).add.mock.calls[0][0];
+          expect(callbackPassedToAdd).toEqual(callbackPassedToRemove);
```

Comment:
I don't understand these checks. callbackPassedToAdd and callbackPassedToRemove are equal as I see.

Reply:
It is a typo. Fixed in https://github.com/DevExpress/DevExtreme/pull/16330/commits/fcb3408ec98d8bb1cbf5cac06409c49357272be7